rllib/utils/sgd.py

```diff
@@ -48,6 +48,12 @@ def minibatches(samples: SampleBatch, sgd_minibatch_size: int, shuffle: bool = T
     if "state_in_0" not in samples and "state_out_0" not in samples:
         samples.shuffle()
 
+    assert (
+        samples.__len__ >= sgd_minibatch_size
```

Comment:
this should be __len__(). a lot of tests are failing.
also, why are you calling __len__() directly? shouldn't you use len(samples) instead?

Reply:
Sorry @gjoliver, but I cannot remember the reason why I didn't use `len(samples)`. But I agree, it should also work with `len(samples)` and I've made an update.